src/main/charts/confluence/values.yaml

```diff
@@ -183,6 +183,18 @@ synchrony:
     periodSeconds: 1
     # -- The number of consecutive failures of the Synchrony container readiness probe before the pod fails readiness checks
     failureThreshold: 30
+  resources:
+    jvm:
+      # -- The maximum amount of heap memory that will be used by the Synchrony JVM
+      maxHeap: "1g"
```

Comment:
I think we should have it at 2Gb to keep consistent with how Confluence creates Managed Synchrony by default. Please see [this file (internal repo)](https://stash.atlassian.com/projects/CONFSERVER/repos/confluence/browse/confluence-plugins/confluence-bundled-plugins/synchrony-interop/src/main/resources/env/synchrony.properties#7) with default properties and [this ticket (public Jira)](https://jira.atlassian.com/browse/CONFSERVER-58401) for default heap size change.